speakeasy/winenv/api/usermode/kernel32.py

```diff
@@ -5567,3 +5567,158 @@ def AddVectoredExceptionHandler(self, emu, argv, ctx={}):
         emu.add_vectored_exception_handler(First, Handler)
 
         return Handler
+
+    @apihook("GetSystemDefaultUILanguage", argc=0)
+    def GetSystemDefaultUILanguage(self, emu, argv, ctx={}):
+        '''
+        LANGID GetSystemDefaultUILanguage();
+        '''
+        return int("0x0409", 16) #"English (United States)"
```

Comment:
thanks for the comments here

Reply:
```suggestion
        return 0x409 #"English (United States)"
```

why not return the int directly?

Reply:
maybe add `LANG_EN_US = 0x409` around like 59 and use that constant here